### DISCUSSION TITLE
Paypal Commerce Account Creation/Linking Issues

### DISCUSSION THREAD

**Discussion by dmayott** [swellstores/community#315] [Help]
We're trying to migrate our existing Squarespace site to use Swell as our commerce provider. When trying to enable Paypal payments, it seems that the sandbox.paypal.com log in is NOT the same as the business account we currently have set up. I made a new account there with the same billing email, and put in all of the shareholder information, etc. It now says we should receive an email to confirm the account, but it has not come after an hour.

Paypal support is telling us to ask about issues on the Swell side even though that doesn't really make any sense. I figured to ask here anyway to see if anyone has had a similar issue and figured out a fix.

This is all in a test environment as of now, if that affects anything.

**Comment by steven-k1** [comment]
Hey, @dmayott Are you running into any errors on Swell's end, or is the issue just not receiving the account confirmation email from PayPal for the newly created business account?

**Reply by dmayott** [reply]
It doesn't seem to be on swell's end. The reason Paypal wanted me to check here first was because they didn't understand why our existing business account was not allowed to sign in with the link given on Swell's payments page. 

My suspicion is that the new account we made via that link cannot be confirmed by Paypal because it uses the same email as our existing account.

**Reply by steven-k1** [reply]
@dmayott Interesting. I am not too familiar with the difference between the accounts, except that http://www.sandbox.paypal.com/ is for accessing the sandbox PayPal account portal, but I believe this should be connected to a main business account.

When I go through the connection process in my personal store between Swell and PayPal I am brought to https://www.sandbox.paypal.com/ and use the same login credentials I always use to access PayPal and it creates a successful connection for me.

Not sure if the account you are trying to connect with originally has a different setup though.

**Reply by dmayott** [reply]
If I try to connect with the existing Paypal business account, I get a error popup saying `Some of your info isn't correct. Please try again.`. If I use the same exact credentials on Paypal directly, it logs in fine.

**Reply by dmayott** [reply]
I re-opened the case with Paypal support because I am fully convinced it is an issue on their end. I will update the thread with what I can get from them. Thanks!

**Reply by steven-k1** [reply]
@dmayott Sounds good. Look forward to hearing what they say.
